contracts/Profile.sol

```diff
@@ -15,53 +15,53 @@ contract Profile {
     event ProfileCreated(
         uint96 indexed identityId,
         address indexed identityContractAddress,
-        bytes indexed nodeId,
+        bytes indexed nodeId
     );
     event StakeIncreased(
         uint96 indexed identityId,
         address indexed identityContractAddress,
```

Comment:
we no longer have identityContractAddress right?

One thing we should do, is emit if this was delegated stake or not. Can be also part of a different event